src/main/java/org/fundacionjala/sfdc/util/CommonActions.java

```diff
@@ -205,10 +239,6 @@ public static void closeMessageLighting() {
                 WEB_DRIVER.findElement(By.id("lexNoThanks")).click();
                 WEB_DRIVER.findElement(By.id("tryLexDialogX")).click();
             }
-        } catch (NoSuchElementException e) {
-            System.out.println("Message not displayed.");
-        }
+        } catch (NoSuchElementException e) { }
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00108)